app/views/prison/dashboards/_request_details_row.html.erb

```diff
@@ -1,24 +1,17 @@
 <tr class="hidden-row hide" aria-hidden="true" aria-expanded="false">
   <td colspan="9" class="request-details">
     <div class="request-row-details">
-      <% if visit.slot_granted? -%>
+      <% visit.slots.each_with_index do |slot, i| %>
```

Comment:
What about collection rendering? Then shoving the inside of the loop in a partial:
```ruby
<%= render visit.slots %>
```